packages/visual-editor/src/components/Draggable/canvasToolsUtils.ts

```diff
@@ -0,0 +1,167 @@
+import type {
+  ComponentDefinition,
+  CompositionComponentNode,
+  CompositionComponentPropValue,
+  CompositionDataSource,
+  CompositionUnboundValues,
+  CompositionVariableValueType,
+  Link,
+} from '@contentful/experience-builder-core/types';
+import { ASSEMBLY_DEFAULT_CATEGORY } from '@contentful/experience-builder-core/constants';
+
+export type ExperienceDataSource = CompositionDataSource;
+export type ExperienceUnboundValues = CompositionUnboundValues;
+export type ExperienceProperty = CompositionComponentPropValue;
+export type ExperiencePropertyLiteral = CompositionVariableValueType;
+
+export type TreeNode = {
```

Comment:
The `TreeNode` type only seems to be used by `getParentNodes()` which isn't used anywhere from what I can tell.  Maybe all of this can be trimmed out?